### ISSUE TITLE
How  to train on something dataset?

### ISSUE THREAD

**Issue by kmyfoer** [mzolfaghari/ECO-efficient-video-understanding#33]
I basically reproduce the result on hmdb51,  but I can't reproduce same result on something dataset.
Can you tell me how to set dropout, learning rate or other training trick?

Thank u very much！

**Comment by TwinkleYR** [comment]
I have finished the steps before make && make install,when I conducted make && make install,it had a error about
No implicit rule found for 'install'.
Finished prerequisites of target file 'install'.
Must remake target 'install'.
make: *** No rule to make target 'install'。 停止。
Can you help me solve the problem?Thank you in advance

**Comment by mzolfaghari** [comment]
@kmyfoer we set the dropout for this dataset to 0.3. You can find the information regarding SomethingSomething dataset [here](https://github.com/mzolfaghari/ECO-efficient-video-understanding/tree/master/models_ECO_Full/something_something). Using the same setting (Caffe version), you should be able to reproduce the results of the paper.
Please let me know if had further questions.

Best,

**Comment by ntuyt** [comment]
I also only achieve around 40 accuracy in something dataset using the provided caffe model.
It is worse than reported 42.4.
Is it because of the difference of the data? Does the author use the frames of something dataset downloaded from the official website?
